save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/BackendAgentRepository.kt

```diff
@@ -36,7 +36,7 @@ class BackendAgentRepository(
 
     override fun getNextTestBatch(containerId: String): Mono<TestBatch> = webClientBackend
         .get()
-        .uri("/agents/get-next-test-batch?agentId=$containerId")
```

Comment:
Could you please check sandbox also

Reply:
Sandbox also uses this API

Reply:
I will.

Reply:
Resolving for now, so that the change propagates into `master` quickly.